Doc/library/unittest.mock.rst

```diff
@@ -2501,10 +2501,10 @@ attributes on the mock that exist on the real class:
 
     >>> from urllib import request
     >>> mock = Mock(spec=request.Request)
-    >>> mock.assret_called_with
+    >>> mock.assert_called_with
     Traceback (most recent call last):
      ...
-    AttributeError: Mock object has no attribute 'assret_called_with'
+    AttributeError: Mock object has no attribute 'assert_called_with'
```

Comment:
> you can only access attributes on the mock that exist on the real class:

Intended typo

Reply:
my bad I though the mock in that case did not include the assert method